doc/source/user_guide/adaptive_templates.rst

```diff
@@ -428,6 +402,39 @@ the function ``draw_objects_debug()`` can be used to plot the regions:
 
     Plot of rotor pocket regions drawn using the ``draw_objects()`` function.
 
+Adding imported DXF geometries to adaptive templates
+****************************************************
+Custom geometry can be imported to Motor-CAD from a DXF file. For information on how to import
+custom geometry from a DXF file, please see the "Custom Machine Geometries" tutorial supplied with
+Motor-CAD.
+
+Once a custom geometry has been imported, it is automatically separated into regions, which will
+appear under **Import** in the **Geometry -> Editor** tab. These imported geometry regions can be
+accessed using the ``get_region_dxf()`` method from the ``ansys.motorcad.core`` API. The geometry
+regions that are currently set in the Motor-CAD model are shown under **Template** in the tree.
+
+.. figure:: ../images/Adaptive_Geometry_DXF__1.png
+    :width: 500pt
+
+    **Geometry -> Editor -> Geometry** tab with an imported custom geometry from DXF file.
+
+By default, the imported regions are not displayed. To display an imported region, select the
+checkbox.
+
+Imported regions are not automatically set in the Motor-CAD model or used by the Motor-CAD
+calculations. Adaptive templates can be used to customise the Motor-CAD model geometry with the
+imported geometry.
+
+Use the ``get_region_dxf()`` method in an adaptive templates script to access an imported region.
+Then, modify and interact with the region in the same way as for any other region object.
+The name, properties and parent regions of the imported region can be defined.
+
+To replace an existing region with an imported DXF region, use the ``Region.replace()`` method.
```

Comment:
Is this a method or function?

Reply:
It is a method as it requires a Region class object